gnuradio-runtime/swig/message.i

```diff
@@ -63,6 +69,8 @@ namespace gr {
   long message_ncurrently_allocated();
 }
 
+
+
```

Comment:
Mistaken whitespace diff?

Reply:
Yes, will remove. A lot of copying around chunks of code has happened.